packages/vaex-core/vaex/dataset.py

```diff
@@ -5053,10 +5053,14 @@ def __call__(self, *expressions, **kwargs):
     def echo(self, arg): return arg
 
     def __array__(self, dtype=None):
-        """Casts the dataset to a numpy array
+        """Gives a full memory copy of the dataset into a 2d numpy array of shape (n_rows, n_columns).
+        Note that the memory order is fortran, so all values of 1 column are contiguous in memory for performance reasons.
 
-        Example:
-        >>> ar = np.array(ds)
+        Note this returns the same result as:
+
+        >>> np.array(ds)
+
+        If any of the columns contain masked arrays, the masks are ignored (i.e. the masked elements are returned as well).
```

Comment:
My point was, whatever we say here, should be reflected in a unittests, now we are free to change this without hitting a failure at the unittests.